packages/config/config/devices/0x027a/zse29.json

```diff
@@ -16,6 +16,244 @@
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"3": {
+			"$if": "firmwareVersion >= 2",
+			"label": "Motion Sensitivity",
+			"description": "Adjust motion sensitivity where 10 is the most sensitive setting.",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 10,
+			"defaultValue": 10,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"4": {
+			"$if": "firmwareVersion >= 2",
+			"label": "Brightness Threshold",
+			"description": "The sensor will report motion only if brightness level is below the set value (10 is the lowest value the sensor reports).",
+			"unit": "lux",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 900,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Set by knob on sensor",
+					"value": 0
+				},
+				{
+					"label": "Ignore brightness",
+					"value": 1
+				},
+				{
+					"label": "10",
+					"value": 10
+				},
+				{
+					"label": "50",
+					"value": 50
+				},
+				{
+					"label": "100",
+					"value": 100
+				},
+				{
+					"label": "200",
+					"value": 200
+				},
+				{
+					"label": "300",
+					"value": 300
+				},
+				{
+					"label": "900 (Test mode)",
+					"value": 900
+				}
+			]
+		},
+		"5": {
+			"$if": "firmwareVersion >= 2",
+			"label": "Motion Re-trigger Time",
+			"description": "Delay before the sensor reports no motion after detecting the last motion activity.",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 720,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Set by knob on sensor",
+					"value": 0
+				},
+				{
+					"label": "5 (Test mode)",
+					"value": 5
+				},
+				{
+					"label": "30",
+					"value": 30
+				},
+				{
+					"label": "60",
+					"value": 60
+				},
+				{
+					"label": "300",
+					"value": 300
+				},
+				{
+					"label": "600",
+					"value": 600
+				},
+				{
+					"label": "720",
+					"value": 720
+				}
+			]
+		},
+		"6": {
+			"$if": "firmwareVersion >= 2",
+			"label": "Lux Reporting Frequency",
+			"unit": "minutes",
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 1440,
+			"defaultValue": 30,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"7": {
+			"$if": "firmwareVersion >= 2",
+			"label": "Manual Motion Re-trigger Time Setting",
+			"description": "Read the value for the Motion Re-trigger Time set by knob on sensor (overridden by setting Motion Re-trigger Time parameter).",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 5,
+			"maxValue": 720,
+			"defaultValue": 5,
+			"readOnly": true,
+			"writeOnly": false,
+			"allowManualEntry": false,
```

Comment:
For read-only parameters, please get rid of the options unless it adds a label. So here keep only the test mode one, but change the label to "Test mode".

Reply:
I think it's important to include the number (of seconds, for instance) in the label with "Test Mode".  "Test Mode" just specifies that the device is in test mode (obviously), but it also means that the re-trigger time is 5 seconds.

Reply:
You missed the parameter above this with the same options. 

If it's important at least make it clearer. "Test mode (Retrigger for 5 seconds)" or something. No one will understand what the 5 means.

Reply:
Not sure if you were looking at the most-recent file; I removed all options except for the "Test mode" option for each read-only parameter.  I *did* keep in the corresponding numerical options for parameters 4 and 5 (although the user can enter them directly), because these correspond to the options that can be set by the knobs on the device.  It's a style point; happy to remove them.

Reply:
Aaaah, strike that.  Knocked out the numerical options; also added a change with how parameter 5 is defined based on the Zooz documentation.